sdk/eventhub/eventhubs-checkpointstore-table/test/tables-checkpointstore.spec.ts

```diff
@@ -0,0 +1,518 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import chai from "chai";
+import * as dotenv from "dotenv";
+const should = chai.should();
+import { TableCheckpointStore } from "../src";
+import debugModule from "debug";
+const debug = debugModule("azure:event-hubs:partitionPump");
+import { Checkpoint, PartitionOwnership } from "@azure/event-hubs";
+import { EnvVarKeys, getEnvVars } from "./utils/testUtils";
+import { TableServiceClient, AzureNamedKeyCredential, TableClient } from "@azure/data-tables";
+import { CheckpointEntity, PartitionOwnershipEntity } from "../src/tableCheckpointStore";
+
+const env = getEnvVars();
+dotenv.config();
+
+/* test to show that test framework is set up well */
+describe("TableCheckpointStore", () => {
+  it("is exported from the package", () => {
+    should.exist(TableCheckpointStore, "Expected TableCheckpointStore to be exported.");
+  });
+});
+
+const service = {
+  storageConnectionString: env[EnvVarKeys.STORAGE_CONNECTION_STRING],
+  storageAccountName: env[EnvVarKeys.STORAGE_ACCOUNT_NAME],
+  storageAccountKey: env[EnvVarKeys.STORAGE_ACCOUNT_KEY],
+  tableName: env[EnvVarKeys.TABLENAME]
```

Comment:
It doesn't look like you use `tableName` or `storageConnectionString`.

Reply:
cool. removing them